packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -20,6 +21,144 @@ import 'scroll_position.dart';
 import 'scroll_position_with_single_context.dart';
 import 'scrollable.dart';
 
+/// Class that supplies children for the [ListWheelScrollView].
+///
+/// Instead of providing a concrete list of children to [ListWheelScrollView],
+/// this class allows several methods to lazily provide children, which is more
+/// efficient when there are too many or infinite children, by bringing in
+/// only the children that are visible in viewport, and remove them when they
+/// are no longer visible.
+///
+/// See also:
+///  * [ListWheelChildListDelegate], a delegate that supplies children using an
+///    explicit list.
+///  * [ListWheelChildLoopingListDelegate], a delegate that supplies infinite
+///    children by looping an explicit list.
+///  * [ListWheelChildBuilderDelegate], a delegate that supplies children using
+///    a builder callback.
+abstract class ListWheelChildDelegate {
+  /// Return the child at the given index. If the child at the given
+  /// index does not exist, return null.
+  Widget build(BuildContext context, int index);
+
+  /// Returns an estimate of the number of children this delegate will build.
+  int get estimatedChildCount;
+
+  /// Returns the true index for a child built at a given index. Defaults to
+  /// the given index, however if the delegate is [ListWheelChildLoopingListDelegate],
+  /// this value is the index of the true element that the delegate is looping to.
+  int trueIndexOf(int index) => index;
+
+  /// Called to check whether this and the old delegate are actually 'different',
+  /// so that the caller can decide to rebuild or not.
+  bool shouldRebuild(covariant ListWheelChildDelegate oldDelegate);
+}
+
+/// Class that supplies children using an explicit list. Children are provided
+/// in the range [0, list.length - 1], outside of that range a null child will
+/// be returned.
+class ListWheelChildListDelegate extends ListWheelChildDelegate {
```

Comment:
This documentation will go up on docs.flutter.io.  Because of the site's search feature, you have no way of knowing which one of these Delegate classes someone searching for will find first.

To make the documentation more accessible, consider duplicating the comment from the first class, ListWheelChildDelegate.  After copying it, change the comment to explain the unique logic just as clearly as you did above.

Reply:
fixed.
Please let me know if it's clean and clear enough.